pkg/perf/healthcheck/healthcheck.go

```diff
@@ -60,30 +58,27 @@ func (h *healthcheckTask) Description() string {
 // Run executes the health checks.
 func (h *healthcheckTask) Run(ctx context.Context) (interface{}, error) {
 	log.Debug().Msg("starting health check task")
-	for k := range h.errors {
-		// reset errors on each run
-		h.errors[k] = nil
-	}
+	errs := make(map[string][]string)
 
 	for label, check := range h.checks {
 		errors := check(ctx)
 		if len(errors) == 0 {
 			continue
 		}
 		stringErrs := errorsToStrings(errors)
-		h.errors[label] = append(h.errors[label], stringErrs...)
+		errs[label] = append(errs[label], stringErrs...)
```

Comment:
I think this can just be 
```suggestion
		errs[label] = stringErrs
```
since errs[label] will always be nil.